### ISSUE TITLE
Testing on NuScenes Validation

### ISSUE THREAD

**Issue by StrivedTye** [Ghostish/Open3DSOT#39]
Hi authors,

Great work!

I have a question about the testing results on NuScenes. In the paper "V2B", they also evaluate the trackers using the keyframes of the validation set, but you have a different number of the final testing frames. For example, you have 64159 cars, V2B has only 15578 cars. Could you give some tips if you know the reason behind it?  

Thanks!

**Comment by Ghostish** [comment]
For V2B, it converts the NuScenes data to KITTI format. Since NuScenes and KITTI have different categories, there may exist some mismatchs between the classes.  Furthermore, we found that V2B needs extra meta data from "nuScenes-lidarseg" to do the data conversion. It probably filters out more frames that were not covered by nuScenes-lidarseg. 

The splits generated by V2B may be just sub-splits of ours.

 As the conversion is not recommended by the NuScenes official, we filter training/testing tracklets directly using official NuScenes APIs without extra meta data from nuScenes-lidarseg.
